docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -1688,6 +1688,31 @@ <h1>Samza Configuration Reference</h1>
                     </td>
                 </tr>
 
+                <tr>
+                    <td class="property" id="stores-side-inputs">stores.<span class="store">store-name</span>.side.inputs</td>
+                    <td class="default"></td>
+                    <td class="description">
+                        Samza applications with large stores that are periodically populated by a secondary data sources such as HDFS, but otherwise ready-only;
+                        Side inputs feature makes it easier and efficient to populate data. It eliminates the need for <span class="stream">changelog stream</span>
+                        and uses the source stream instead to bootstrap the store in case of failure thereby, reducing additional copy of the data. The source stream
+                        is marked as a <a href="../container/streams.html#bootstrapping">bootstrap stream</a> and once the store is bootstrapped, it is updated in
+                        change capture mode in the background. The value is a comma-separated list of streams that are side inputs to the store.
+                        Each stream is of the format <span class="system">system-name</span>.<span class="stream">stream-name</span>. Applications are
+                        required to add the side inputs to <a href="#task-inputs" class="property">task.inputs</a> as well.
```

Comment:
iirc, the new configurations are hosted in a different place?  cc: @dxichen

Reply:
@bharathkk can you please also update the `samza-configurations.md` file?